src/shared/lib/StoreUtils.spec.ts

```diff
@@ -498,4 +499,16 @@ describe('StoreUtils', () => {
                 "fetchStudy should be called with the correct study id (study4)");
         });
     });
+
+    describe('wholeGenomeProfiled()', () => {
```

Comment:
`describe('noGenePanelUsed` ?

Reply:
Ha ha good catch; unit test name was not updated after rename of method name.